index.js

```diff
@@ -1,30 +1,51 @@
+const multiRet = ['push', 'pop', 'shift', 'unshift']
```

Comment:
That's strange! Not what I would expect at all. Maybe it is down to babili being in beta. If that's the case then we can leave the change to `const` as-is. 👍